### ISSUE TITLE
End of file newline is removed

### ISSUE THREAD

**Issue by fappelman** [vknabel/vscode-swiftformat#13]
I have a simple example file `demo.swift`:

```swift
import Foundation

struct Demo {
    init() {
        // Bla
    }
}
```
It is important to notice that the file does end with a newline:

```bash
$ od -c demo.swift
0000000    i   m   p   o   r   t       F   o   u   n   d   a   t   i   o
0000020    n  \n  \n   s   t   r   u   c   t       D   e   m   o       {
0000040   \n                   i   n   i   t   (   )       {  \n
0000060                            /   /       B   l   a  \n
0000100        }  \n   }  \n
0000105
```

When I run `swiftformat` using the command line:

```bash
$ swiftformat --verbose --config ~/.config/swiftformat/config demo.swift
Running SwiftFormat...
Formatting demo.swift
-- no changes (cached)

SwiftFormat completed in 0.01s.
0/1 files formatted.
```
As you can see `swiftformat` is fine with this demo struct. 

The next step is then to run this within `VSCode`. I have configured the exact same config file and would expect the same output. It is not, it removes the last line.

When I save it I get the following content with `od`:

```bash
$ od -c demo.swift
0000000    i   m   p   o   r   t       F   o   u   n   d   a   t   i   o
0000020    n  \n  \n   s   t   r   u   c   t       D   e   m   o       {
0000040   \n                   i   n   i   t   (   )       {  \n
0000060                            /   /       B   l   a  \n
0000100        }  \n   }
0000104
```

That new line should not be removed.

**Comment by fappelman** [comment]
As a workaround I have enabled `Insert Final Newline` which restores the missing newline but that should not be the solution.

**Comment by vknabel** [comment]
@fappelman Good catch! Thanks for the great report.
I will have a deeper look soon!

**Comment by vknabel** [comment]
Thanks again. Fix released as [1.3.4](https://github.com/vknabel/vscode-swiftformat/releases/tag/1.3.4)

**Comment by fappelman** [comment]
I did notice that the other package had the same issue. Thanks for fixing.
